app/models/operations/load_service_areas.rb

```diff
@@ -103,7 +109,11 @@ def create_records(input)
           Failure({message: "#{e}"})
         end
       end
-      Success({message: "Successfully created/updated #{input[:result].size} Service Area records"})
+      if input[:result].size > 0
```

Comment:
We don't really want to do this. Expectation is to just return total number of records imported and proceed with next step. A failure meant to stop the process entirely.

Reply:
I have removed the failure case, please check